language/debug/config.go

```diff
@@ -17,6 +18,21 @@ func (dl *debugLang) RegisterFlags(fs *flag.FlagSet, cmd string, c *config.Confi
 	dc := newDebugConfig()
 	c.Exts[DebugLangName] = dc
 
+	// check for env vars
+	if logLevel, ok := os.LookupEnv("GAZELLE_LOG_LEVEL"); ok {
+		level, err := zerolog.ParseLevel(logLevel)
+		if err != nil {
+			fmt.Printf("warning: bad log_level: %v", err)
+		} else {
+			dc.Logger = dc.Logger.Level(level)
+		}
+	}
+	if progress, ok := os.LookupEnv("GAZELLE_PROGRESS"); ok {
+		if progress == "true" || progress == "1" {
+			dc.showTotalElapsedTimeMessages = true
+		}
+	}
+
 	dl.start = time.Now()
 	dl.prev = time.Now()
```

Comment:
```suggestion
	now := time.Now()
	dl.start = now
	dl.prev = now
```
This way, they don't drift.